docs/src/CommutativeAlgebra/GroebnerBases/groebner_bases.md

```diff
@@ -71,10 +71,7 @@ julia> default_ordering(F)
 degrevlex([x, y, z])*lex([gen(1), gen(2)])
 
 julia> S, _ = grade(R, [1, 2, 3])
-(Multivariate polynomial ring in 3 variables over QQ graded by
-  x -> [1]
-  y -> [2]
-  z -> [3], MPolyDecRingElem{QQFieldElem, QQMPolyRingElem}[x, y, z])
+(Graded multivariate polynomial ring in 3 variables over QQ, MPolyDecRingElem{QQFieldElem, QQMPolyRingElem}[x, y, z])
```

Comment:
how?

Reply:
'... over QQ with variables [x,y,z]'

Reply:
What we print here is a `tuple (Ring, Vector{Variables})  `  because the constructor also returns the variables.
Arrays always print their type. Since we do not own arrays (they come from `Base`), we cannot change this. 
And it might not be a good idea even if we could.

Reply:
I see.
What struck me, was the enormous disbalance between the amount of detail printed for the first entry (nice and suitable for mathematicians) and the second entry (which easily takes more than a line before even opening the square bracket, if you are dealing with a graded ring over an algebraic extension of QQ).

Reply:
Something very similar has been discussed in https://github.com/oscar-system/Oscar.jl/issues/2064, although without a result.